src/pages/components/dropdown/usage.mdx

```diff
@@ -152,111 +167,449 @@ Another important difference between the two components is the underlying code.
+
+<Row>
+<Column colLg={8}>
+
+![Dropdown overflow text truncation.](images/dropdown-usage-6.png)
+
+</Column>
+</Row>
+
+### Further guidance
+
+For further content guidance, see Carbon's [content guidelines](notion://www.notion.so/guidelines/content/general).
+
+## Universal behaviors
+
+The behaviors listed in this section are universal across all of the types. For behaviors that are unique to each type, see the sections below.
+
+### Direction
+
+A dropdown can open up or down depending on its position on the screen. For example, if the dropdown appears at the bottom and close to the edge of the interface, the menu expands upward to avoid being cropped. By default, our dropdowns open downward.
+
+### Elevation
+
+Carbon has one layer style called a `box-shadow`. The box-shadow is placed behind the dropdown menu when open. Box-shadow is also used in other components that have overlayed menus, such as the overflow menu and date picker calendar. The SCSS for box-shadow is `0 2px 6px 0 rgba(0,0,0,.2)`.
+
+### Scrolling behavior
+
+Scroll bars may not always be enabled so we recommend showing 50% of the container height of the last option before scrolling to indicate there is more to see within the menu. We recommend starting a scroll at the fifth option in the menu list, but this may vary based on your specific use case.
+
+<Row>
+<Column colLg={8}>
+
+![Dropdown scrolling menu option cutoff.](images/dropdown-usage-7.png)
+
+</Column>
+</Row>
+
+### Interactions
+
+#### Mouse
+
+- Users trigger a `dropdown` menu to open or close by clicking the chevron icon or clicking anywhere within the field. _Note: You can only open or close a combobox by clicking the chevron icon._
+
+<Row>
+<Column colLg={8}>
+
+![Dropdown closed and open state click targets.](images/dropdown-usage-8a.png)
+
+</Column>
+</Row>
+
+- To select an option the user can click anywhere inside an option container.
+
+<Row>
+<Column colLg={8}>
+
+![Dropdown menu option click target.](images/dropdown-usage-8b.png)
+
+</Column>
+</Row>
+
+- To clear all selected options from a list in a `multiselect dropdown`, click the “x” icon next to the value inside the tag.
+
+<Row>
+<Column colLg={8}>
+
+![Multiselect dropdown tag clear click target.](images/dropdown-usage-8c.png)
+
+</Column>
+</Row>
+
+- To clear a selected value in a `combobox` or a filterable dropdown, click the “x” icon to the right of the field input text.
+
+<Row>
+<Column colLg={8}>
+
+![Combobox and filterable dropdown clear click target.](images/dropdown-usage-8d.png)
+
+</Column>
+</Row>
+
+#### Keyboard
+
+#### Dropdown:
+
+- The dropdown field receives focus if none of the options are selected in the menu.
+- Users can move focus to the next option by pressing the `Down Arrow`.
+- Users can move focus to the previous option by pressing the `Up Arrow`.
+- Users can open the dropdown menu by pressing the `Spacebar`, `Return`, `Down arrow` or `Up Arrow`.
```

Comment:
Should we use `Enter` instead of `Return`? Thinking about how WAI ARIA documents stuff over in: https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

Reply:
Yes you are right, we should use `Enter` instead. 👍🏻 We are doing this for other components.